src/main/java/com/faforever/client/preferences/ChatPrefs.java

```diff
@@ -97,14 +95,6 @@ public void setTimeFormat(TimeInfo time) {
     this.timeFormat.set(time);
   }
 
-  public DateInfo getDateFormat() {
-    return dateFormat.get();
-  }
-
-  public void setDateFormat(DateInfo date) {
-    this.dateFormat.set(date);
-  }
-
```

Comment:
Would be nice if we kept these for one or two release and use them in the migratePreferences method of the preferences service so people don't lose the settings

Reply:
Do you use some sort of convention to mark things as deprecated in the client?

Reply:
@ Deprecated